sklearn/ensemble/stacking_classifier.py

```diff
@@ -0,0 +1,319 @@
+                                     "the default `auto`" % estimator)
+        else:
+            method = self.method
+        return method
+
+    def fit(self, X, y, **kwargs):
+        """ Fit the estimators.
+
+        Parameters
+        ----------
+        X : {array-like, sparse matrix}, shape = [n_samples, n_features]
+            Training vectors, where n_samples is the number of samples and
+            n_features is the number of features.
+
+        y : array-like, shape = [n_samples]
+            Target values.
+
+        **kwargs: optional fit parameters
+
+
+        Returns
+        -------
+        self : object
+        """
+        if any(s in type_of_target(y) for s in ['multilabel', 'multioutput']):
+            raise NotImplementedError('Multilabel and multi-output'
+                                      ' classification is not supported.')
+
+        if not self.estimators:
+            raise AttributeError('Invalid `estimators` attribute %s, '
+                                 '`estimators` should be a list of (string, '
+                                 'estimator) tuples.' % type(self.estimators))
+
+        if not is_classifier(self.meta_estimator):
+            raise AttributeError('Invalid `meta_estimator` attribute '
+                                 '%s, `meta_estimator` should be a classifier.'
+                                 % type(self.meta_estimator))
+        for name, step in self.estimators:
+            if not hasattr(step, self._method(step)):
+                raise ValueError('Underlying estimator %s %s does '
+                                 'not support %s.' % (name,
+                                                      type(step),
+                                                      self._method(step)))
+
+        for param in kwargs:
+            if not has_fit_parameter(self.meta_estimator, param):
+                raise ValueError('Underlying meta estimator %s '
+                                 'does not support `%s`.' % (
+                                     type(self.meta_estimator), param))
+            for name, step in self.estimators:
+                if not has_fit_parameter(step, param):
+                    raise ValueError('Underlying estimator %s of type %s does '
+                                     'not support `%s`.' % (name,
+                                                            type(step),
+                                                            param))
+
+        self.le_ = LabelEncoder().fit(y)
+        self.classes_ = self.le_.classes_
+
+        transformed_y = self.le_.transform(y)
+        if self.cv == 1:  # Do not cross-validation
+            # Parallel fit each estimator
+
+            self.estimators_ = Parallel(n_jobs=self.n_jobs)(
+                    delayed(_parallel_fit)(clone(clf),
+                                           X, transformed_y, kwargs)
+                    for _, clf in self.estimators)
+            scores = self._est_predict(X)
```

Comment:
I don't feel `scores` is the best name. Perhaps `y_pred` or `y_score` or `predictions` or `Xt`